dataland-frontend/tests/e2e/specs/admin-tools/RabbitMQAdmin.ts

```diff
@@ -24,27 +23,41 @@ const queues = [
   'community-manager.queue.nonSourceableData',
 ];
 
+/**
+ * Checks if all expected queues exist.
+ * @param $rows - rows of the table.
+ * @param expectedQueues - expected queues.
+ */
+function checkQueuesExist($rows: JQuery<HTMLElement>, expectedQueues: string[]): void {
+  const actualQueues = [...$rows].flatMap((row) => {
+    const nameCell = row.querySelector('td:nth-child(2) a');
+    const name = nameCell?.textContent?.trim();
+    return name ? [name] : [];
```

Comment:
Expliziter Return Type fuer die flatMap sowie 2 Zeilen verbunden.
```suggestion
  const actualQueues = [...$rows].flatMap((row): string[] => {
    const name = row.querySelector('td:nth-child(2) a')?.textContent?.trim();
    return name ? [name] : [];
```

Reply:
implemented